/**:
  ros__parameters:
    Bot1_YellowA_config: "bot1_yellow_a.xml"
    Bot1_YellowB_config: "bot1_yellow_b.xml"
    Bot1_YellowC_config: "bot1_yellow_c.xml"
    Bot1_YellowD_config: "bot1_yellow_d.xml"
    Bot1_YellowE_config: "bot1_yellow_e.xml"
    Bot1_YellowS_config: ""
    Bot1_BlueA_config: "bot1_blue_a.xml"
    Bot1_BlueB_config: "bot1_blue_b.xml"
    Bot1_BlueC_config: "bot1_blue_c.xml"
    Bot1_BlueD_config: "bot1_blue_d.xml"
    Bot1_BlueE_config: "bot1_blue_e.xml"
    Bot1_BlueS_config: "template_bot1_blue_a.xml"
    Bot2_YellowA_config: "bot2_yellow_a.xml"
    Bot2_YellowB_config: "bot2_yellow_b.xml"
    Bot2_YellowC_config: "bot2_yellow_c.xml"
    Bot2_YellowD_config: ""
    Bot2_YellowE_config: ""
    Bot2_YellowF_config: ""
    Bot2_YellowS_config: ""
    Bot2_BlueA_config: "bot2_blue_a.xml"
    Bot2_BlueB_config: "bot2_blue_b.xml"
    Bot2_BlueC_config: "bot2_blue_c.xml"
    Bot2_BlueD_config: ""
    Bot2_BlueE_config: ""
    Bot2_BlueF_config: ""
    Bot2_BlueS_config: ""
    tree_node_model_config_file: "/Eurobot-2025-Main/src/bt_app_2025/bt_m_config/bt_m_tree_node_model.xml"
    groot_xml_config_directory: "/Eurobot-2025-Main/src/bt_app_2025/bt_m_config/"
